src/main/java/com/untamedears/ItemExchange/utility/ItemExchange.java

```diff
@@ -275,23 +270,14 @@ public void playerResponse(Player player, ItemStack itemStack, Location location
 			IETransactionEvent event = new IETransactionEvent(player, location, playerInput, exchangeOutput);
 			Bukkit.getPluginManager().callEvent(event);
 
-			// Power buttons.
-			Block block = location.getBlock();
-			ItemExchange.powerBlock(player, block);
-
-			Material type = block.getType();
-			if(type == Material.CHEST || type == Material.TRAPPED_CHEST) {
-				Block north = block.getRelative(BlockFace.NORTH);
-				Block south = block.getRelative(BlockFace.SOUTH);
-				Block east = block.getRelative(BlockFace.EAST);
-				Block west = block.getRelative(BlockFace.WEST);
-
-				if(north.getType() == type) ItemExchange.powerBlock(player, north);
-				if(south.getType() == type) ItemExchange.powerBlock(player, south);
-				if(east.getType() == type) ItemExchange.powerBlock(player, east);
-				if(west.getType() == type) ItemExchange.powerBlock(player, west);
-			}
+			// Power buttons button directly behind *this* chest
+			Block ShopChest = location.getBlock();
```

Comment:
Variable name should start with lower case

Reply:
Done.